courses/README.md

```diff
@@ -1,58 +1,119 @@
+  - all necessary dates will be pulled in automatically from xPRO Studio on a nightly basis. Or you can use the management command `sync_courseruns` to do it immediately. 
+  - **Live**: should be unchecked until you're ready to make the course run live. This value determines if the start date will appear in the list of options for the course page in the CMS. 
+  - **Save** and note the courserun id, you will need it later. 
+
   
-4. Create a product, if necessary, at /admin/ecommerce/product/add/
-
-   a courserun without a product isn't much use. Users won't be able to enroll. 
-
-   - set content type to Course Run 
-   - Set object id to the courerun id from the courserun you created. 
-   - Set `is_active` to true. Users cannot checkout with an inactive product
-   - Set `availble_in_bulk_form` as appropriate. Private courses, demos and pilot courses should be set to false
-   - Save the product and note the product id, you will need it later
-   - Products are incomplete without a Product Version, create one at /admin/ecommerce/productversion/add/ 
-     - Enter the product id for the product you just created
-     - Set the price as directed by the course team. 
-     - Set the description to be the same as the courserun title. The product title will appear on the checkout page
-   - test the product by going to /checkout/?product= and the courserun id (text or integer)  
-
-5. Check to make sure the course run in Studio has an appropriate professional track
-
-   - go to /admin/coursemode in xPRO Studio
-   - if a coursemode doesn't already exist for the courserun, add a new one
-    
-     - choose the course run from the drop-down list
-     - choose the mode to "no-id-professional" from the drop-down
-     - Set the name of the coursemode. By convention this is "Professional"
-    
-    
\ No newline at end of file
+5. **Create a product**, if necessary, at `/admin/ecommerce/product/add/` (required for enrollment)
+  - **Content type**: set to Course Run 
+  - **Object id**: set to the courerun id from the course run you created. 
+  - **is_active**: Set `is_active` to true. Note that this needs to be set to check the checkout for the product version (below). If checked, the product start date will appear in the pulldown on the checkout page. You may want to uncheck it after testing until you're ready to launch.
+  - **availble_in_bulk_form**: Set `availble_in_bulk_form` as appropriate. Private courses, demos and pilot courses should be set to false
+  - **Save** the product and note the product id, you will need it later
+
+
+6. **Create a product version**, if created a product, at `/admin/ecommerce/productversion/add/` 
+  - **Product**: Enter the product id for the product you just created
+  - **Price**: Set the price as directed by the course team. 
+  - **Description**: Set the description to be the same as the course run title. The product title will appear on the checkout page
```

Comment:
Out of scope for this PR, but do we have an issue with product titles (displayed on checkout) getting out of sync with course titles (displayed in Studio and BI)?

Reply:
I will investigate.